intake/catalog/base.py

```diff
@@ -434,9 +449,16 @@ def _load(self):
             self._page_size = None
             self._entries._page_cache.update(
                 {source['name']: RemoteCatalogEntry(
-                    url=self.source_url,
+                    url=self.url,
```

Comment:
All of the `__init__`s now expect the base URL instead of a URL that has `/v1/source` or `/v1/info` in it. It makes composition simpler if we always expect the base URL.